include/deal.II/base/bounding_box.h

```diff
@@ -134,8 +136,17 @@ class BoundingBox
    */
   double volume() const;
 
+  /**
+   * Boost serialization function
+   */
+  template<class Archive>
+  void serialize(Archive &ar, const unsigned int /*version*/)
```

Comment:
please separate definition and declaration.

Reply:
(you can still put the declaration in the h file, at the end, enclosed in `#ifndef DOXYGEN ... #endif`).

Reply:
not only you can, but i think you should as it's templated with `Archive` and we can not instantiate it for some a priori known types.